### ISSUE TITLE
Ensure to have a11y text to screen readers

### ISSUE THREAD

**Issue by ShubhamOulkar** [expressjs/expressjs.com#1829]
#### Following HTML element need a11y text
```
 <a id="top" href="#" aria-hidden="true">
   <svg xmlns="http://www.w3.org/2000/svg" width="10" height="6" fill="none" viewBox="0 0 
     1155 1000"><path fill="currentColor" d="m577.344 0 577.346 1000H0L577.344 0Z"></path>
   </svg>
  </a>

<section id="logo">
   <a href="/" class="express">Express</a>
 </section>
```

Actually elements are in tabbing order and does not have accessible text for screen readers. 
 
#### Success criteria 
[WCAG 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html), [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

#### How to fix?
Add aria-label="scroll to top" and role="button" attributes on anchor link.
Add title="Go to Expressjs home" on logo.

**Comment by ShubhamOulkar** [comment]
Hey @bjohansebas  we have logos in images/brand folder. I think we should use them in header.

**Comment by carlosstenzel** [comment]
If you can make the proposed adjustments, you can open a PR for review. Thank you for the analysis.

**Comment by bjohansebas** [comment]
Yep, that's something I've wanted to do before. PRs are welcome.
